light_prop/utils.py

```diff
@@ -0,0 +1,48 @@
+import logging
```

Comment:
I would rather not have script like `utils`. We will start putting everything which does not fit elsewhere here and it will be a mess soon.

This should be `io.image` maybe?

Reply:
Okie, or in light field/io